oncall/readme.md

```diff
@@ -0,0 +1,13 @@
+# Auto Issue Assign Instruction
+## Define Variables
+Make sure to define correct repository variables in repo setting page. Enter github username in `ONCALL_LIST`. Custom oncall info is in `CUSTOM_ONCALL_ROTATION` with the following format:
+
+`"{github-username},{start-date},{end-date}"`
```

Comment:
Are these start/end dates inclusive or exclusive? Trying to understand what we should put if there are 2 custom shifts back-to-back (for example)

Reply:
Yes so the dates are inclusive date. In a back-to-back scenario, if say first person has a end date on March 6 and the second person put March 6 as the start date, it'll assign issues to whichever is on the top of the list.